compiler/rustc_lint/src/builtin.rs

```diff
@@ -595,30 +591,16 @@ impl<'tcx> LateLintPass<'tcx> for MissingDoc {
 
     fn check_item(&mut self, cx: &LateContext<'_>, it: &hir::Item<'_>) {
         match it.kind {
-            hir::ItemKind::Trait(.., trait_item_refs) => {
+            hir::ItemKind::Trait(..) => {
                 // Issue #11592: traits are always considered exported, even when private.
```

Comment:
Looks like this whole arm can be removed now (including this comment).

Reply:
Wouldn't removing this arm will change behaviour?

Reply:
Maybe, but it still looks like a reasonable change.
It can be done in a separate PR.